pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -557,7 +563,7 @@ func (p *PriorityQueue) runPreEnqueuePlugin(ctx context.Context, pl framework.Pr
 // addToActiveQ tries to add pod to active queue. It returns 2 parameters:
 // 1. a boolean flag to indicate whether the pod is added successfully.
 // 2. an error for the caller to act on.
-func (p *PriorityQueue) addToActiveQ(logger klog.Logger, pInfo *framework.QueuedPodInfo) (bool, error) {
+func (p *PriorityQueue) addToActiveQ(logger klog.Logger, pInfo *framework.QueuedPodInfo, onAdd func()) (bool, error) {
```

Comment:
Can we change it to `moveToActiveQ`? and instead of handling other queues in `onAdd`, we always check all queues, remove the pod from there, and push the pod into activeQ.

Reply:
Good idea, done